### PR TITLE
Save search submit to local storage

### PR DESCRIPTION
**Pull request by mocca102**

### CODE REVIEW

File: src/beaconUtils.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
+export const CONSTANTS = {
```
Comment (mocca102) [inline src/beaconUtils.js:1]: Copied from the beacon code

File: src/utils.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {
   Nullable,
   ConstructorClientOptions,
 } from '@constructor-io/constructorio-client-javascript/lib/types';
+import { storageSetItem, storeRecentSearch, storeRecentAction, CONSTANTS } from './beaconUtils';
```
Comment (esezen) [inline src/utils.ts:7]: Lint

File: src/utils.ts [hunk 3]
```diff
@@ -256,3 +257,20 @@ export const translate = (word: string, translations?: Translations) => {
 
   return localTranslations[word] || word;
 };
+export const addTrackSearchSubmit = (cioClient, term, autocompleteData = {}) => {
```
Comment (esezen) [inline src/utils.ts:260]: Naming thing we talked about

**Review by esezen — CHANGES_REQUESTED** [review]

**Review by esezen — APPROVED** [review]
LGTM! Let's get one more review

**Pull request merged**